README.md

```diff
@@ -6,7 +6,7 @@ It also has the ability to inject Vault credentials into a build pipeline or fre
 
 ## Vault Authentication Backends
 This plugin allows authenticating against Vault using the AppRole authentication backend. Hashicorp recommends using AppRole for Servers / automated workflows (like Jenkins) and using Tokens (default mechanism, Github Token, ...) for every developer's machine.
-  Furthermore, this plugin allows using a Github personal access token, or a Vault Token - either configured directly in Jenkins or read from an arbitrary file on the Jenkins Machine.
+  Furthermore, this plugin allows using a Github personal access token, or a Vault Token - either configured directly in Jenkins or read from an arbitrary file on the Jenkins machine.
```

Comment:
The Jenkins community has settled on calling Jenkins controller.
See https://www.jenkins.io/doc/book/glossary/

```suggestion
  Furthermore, this plugin allows using a Github personal access token, or a Vault Token - either configured directly in Jenkins or read from an arbitrary file on the Jenkins controller.
```